catalog/catalog-app/src/main/resources/features.xml

```diff
@@ -264,6 +264,16 @@
         <bundle>mvn:ddf.catalog.admin/catalog-admin-module-sources/${project.version}</bundle>
     </feature>
 
+    <feature name="catalog-transformer-streaming-api" install="manual" version="${project.version}"
+             description="DDF Generic Input Transformer API">
+             <bundle>mvn:ddf.catalog.transformer/catalog-transformer-streaming-api/${project.version}</bundle>
+    </feature>
+
+    <feature name="catalog-transformer-streaming-impl" install="manual" version="${project.version}"
```

Comment:
What is the difference between a feature and a bundle here? I remember from Richard's tech talk that it's good practice to put API and Impl in different bundles. The tech talk didn't address features, though.